External/FEXCore/Source/Interface/Core/OpcodeDispatcher/Vector.cpp

```diff
@@ -38,6 +38,18 @@ void OpDispatchBuilder::MOVAPSOp(OpcodeArgs) {
   StoreResult(FPRClass, Op, Src, -1);
 }
 
+void OpDispatchBuilder::VMOVAPSOp(OpcodeArgs) {
+  OrderedNode *Src = LoadSource(FPRClass, Op, Op->Src[0], Op->Flags, -1);
+  const auto Is128BitDest = GetDstSize(Op) == Core::CPUState::XMM_SSE_REG_SIZE;
+
+  if (Op->Dest.IsGPR() && Is128BitDest) {
+    // Perform 32-bit store to clear the upper lane.
```

Comment:
32-bit or 32-byte?

Reply:
oh whoops, definitely meant byte there. Was probably thinking about 256-bit and 32-byte at the same time and both won out in their own way.